src/bin/avail-light.rs

```diff
@@ -147,6 +162,7 @@ async fn run(error_sender: Sender<anyhow::Error>) -> Result<()> {
 			if !network_dump_event.current_multiaddress.is_empty() {
 				*network_stats_metrics.multiaddress.write().unwrap() =
 					network_dump_event.current_multiaddress;
+				*network_stats_metrics.ip.write().unwrap() = network_dump_event.current_ip;
```

Comment:
Maybe just to put expect here instead of unwrap?

Reply:
Fixed, tnx.